src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -289,49 +278,14 @@ internal override TypeSymbol IteratorElementType
 
         internal override ObsoleteAttributeData ObsoleteAttributeData => null;
 
-        internal override MarshalPseudoCustomAttributeData ReturnValueMarshallingInformation => null;
-
-        internal override CallingConvention CallingConvention => CallingConvention.Default;
-
-        internal override bool HasDeclarativeSecurity => false;
-
-        internal override bool RequiresSecurityObject => false;
-
         public override Symbol AssociatedSymbol => null;
 
-        public override Accessibility DeclaredAccessibility => ModifierUtils.EffectiveAccessibility(_declarationModifiers);
-
-        public override bool IsAsync => (_declarationModifiers & DeclarationModifiers.Async) != 0;
-
-        public override bool IsStatic => (_declarationModifiers & DeclarationModifiers.Static) != 0;
-
-        public override bool IsVirtual => (_declarationModifiers & DeclarationModifiers.Virtual) != 0;
-
-        public override bool IsOverride => (_declarationModifiers & DeclarationModifiers.Override) != 0;
-
-        public override bool IsAbstract => (_declarationModifiers & DeclarationModifiers.Abstract) != 0;
-
-        public override bool IsSealed => (_declarationModifiers & DeclarationModifiers.Sealed) != 0;
-
-        public override bool IsExtern => (_declarationModifiers & DeclarationModifiers.Extern) != 0;
-
-        public bool IsUnsafe => (_declarationModifiers & DeclarationModifiers.Unsafe) != 0;
-
-        internal bool IsExpressionBodied => _syntax.Body == null && _syntax.ExpressionBody != null;
-
-        public override DllImportData GetDllImportData() => null;
-
-        internal override ImmutableArray<string> GetAppliedConditionalSymbols() => ImmutableArray<string>.Empty;
+        internal override bool IsExpressionBodied => _syntax.Body == null && _syntax.ExpressionBody != null;
```

Comment:
>internal override bool IsExpressionBodied => _syntax.Body == null && _syntax.ExpressionBody != null; [](start = 8, length = 100)

It looks like there is an inconsistency between this code and the information passed in constructor initializer above.